projects/igniteui-angular/src/lib/grids/common/events.ts

```diff
@@ -76,22 +79,26 @@ export interface IGridEditDoneEventArgs extends IBaseEventArgs {
      * Indicates if the editing cosists of adding a new row
      */
     isAddRow?: boolean;
-    /** 
+    /**
      * Optional
-     * Indicates if the new value would be valid. 
-     * It can be set to return the result of the methods for validation of the grid 
+     * Indicates if the new value would be valid.
+     * It can be set to return the result of the methods for validation of the grid
      */
     valid?: boolean;
 }
 
-/** 
+
+/**
  * Represents event arguments related to grid editing.
  * The event is cancelable
  * It contains information about the row and the column, as well as the old and nwe value of the element/cell
  */
 export interface IGridEditEventArgs extends CancelableEventArgs, IGridEditDoneEventArgs {
 }
 
+
+export interface IRowDataCancelableEventArgs extends Omit<IGridEditEventArgs, 'rowID' | 'oldValue' | 'newValue' | 'cellID' | 'column'> {}
```

Comment:
Okay, while the exclude is clever and all, its's a bit odd and not really sustainable I think. First we're tying to a interface used for different stuff and not omitting all the junk in it (looking at event, isAddRow, valid and some more) that are not emitted by the -ed counterpart.
If this is to be used only for `rowAdd` and `rowDelete` can we at least just do:
```suggestion
export interface IRowDataCancelableEventArgs extends IRowDataEventArgs, CancelableEventArgs {}
```
That way the events are _actually_ compatible with the previous version and only add the cancelable functionality we wanted?